python/tvm/contrib/utils.py

```diff
@@ -99,7 +99,8 @@ def __init__(self, custom_path=None):
 
         self._created_with_keep_for_debug = self._KEEP_FOR_DEBUG
         if custom_path:
-            os.mkdir(custom_path)
+            if not os.path.exists(custom_path):
+                os.makedirs(custom_path)
```

Comment:
Hi @huanmei9, thanks for the PR. Do you mind clarifying on what is the use case for this change? As @Mousius commented, it is not usually good practice to run tests on a polluted directory, that contains files from previous executions.

Apart from that, as others suggested, here a built-in suggestion ([ref](https://docs.python.org/3/library/os.html#os.makedirs)) that can avoid the need to the `if ...` block on this PR:

```suggestion
            os.makedirs(custom_path, exist_ok=True)
```

But still I'm not sure we want to be blindly reusing directories as temporary directories for tests. Interested to know some other opinions. What do you think?